server/model/taggedBailEditModel.ts

```diff
@@ -56,6 +56,12 @@ export default class TaggedBailEditModel {
     }
   }
 
+  private getCaseDetailsCell(): string {
+    return isSentenceRecalled(this.sentenceAndOffence.sentenceCalculationType)
+      ? `${this.getCourtName()} ${getSentenceRecallTagHTML()}<br>${this.getCaseReference()} ${this.getSentenceDate()}`
```

Comment:
You can put the ternery inside the string to reduce the duplication.

``${this.getCourtName()} ${isSentenceRecalled(this.sentenceAndOffence.sentenceCalculationType) ? getSentenceRecallTagHTML() : ''}<br>${this.getCaseReference()} ${this.getSentenceDate()}`

Reply:
I've updated the code as you suggested